### PR TITLE
e2ee: fix minor issues

### PR DESCRIPTION
**Pull request by fippo**
see individual commits

### CODE REVIEW

File: src/content/peerconnection/endtoend-encryption/css/main.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```
Comment (alvestrand) [inline src/content/peerconnection/endtoend-encryption/css/main.css]: the main.css file is copied from another source, so 2015 is right

File: src/content/peerconnection/endtoend-encryption/index.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```
Comment (alvestrand) [inline src/content/peerconnection/endtoend-encryption/index.html:3]: this one is changed enough that it's reasonable to call it new, even though it's also a branch from another file.

**Review by alvestrand — COMMENTED** [review]
the fork history of the 2 major-changed files can be left behind, but the css is unchanged from its origin.

File: src/content/peerconnection/endtoend-encryption/css/main.css (hunk 1 above)
Comment (fippo) [reply src/content/peerconnection/endtoend-encryption/css/main.css]: we should just bring those common things into main.css. Some day...

Comment (fippo) [reply src/content/peerconnection/endtoend-encryption/css/main.css]: i was tempted to change the css not to include audio but then I assume we'll add audio shortly :-)

**Comment by alvestrand** [comment]
can you remove the CSS file change, so that I can merge the rest of the year fixes?

File: src/content/peerconnection/endtoend-encryption/css/main.css (hunk 1 above)
Comment (fippo) [reply src/content/peerconnection/endtoend-encryption/css/main.css]: this is obsolete but github did not auto-resolve

**Comment by fippo** [comment]
it was already removed, github didn't autoresolve the conversation :-|

**Pull request merged**